RELEASE_NOTES.md

```diff
@@ -7,30 +7,24 @@ Example Note:
 Details about the thing that changed that needs to get included in the Release Notes in markdown.
 -->
 
-# knife-ec2 0.16.0 release notes:
-In this release we have added features for tagging EBS volumes and tagging node in Chef. There are a couple of bug fixes and enhancement as well.
+# knife-ec2 0.17.0 release notes:
+In this release we have added features for add existing eni to server and added `--instance-initiated-shutdown-behavior`. There are a couple of bug fixes and enhancement as well.
```

Comment:
In this release we have added feature to added `--primary-eni` option to add exisiting ENI to server. Added `--instance-initiated-shutdown-behavior` option to stop or terminate the instance on shutdown. There are a couple of bug fixes and enhancement as well.